beekeeper-cleanup/src/main/java/com/expediagroup/beekeeper/cleanup/path/aws/S3SentinelFilesCleaner.java

```diff
@@ -32,7 +32,7 @@ public S3SentinelFilesCleaner(S3Client s3Client) {
 
   @Override
   public void deleteSentinelFiles(String absolutePath) {
-    AmazonS3URI s3Path = new AmazonS3URI(absolutePath, false);
+    AmazonS3URI s3Path = new AmazonS3URI(absolutePath, true);
```

Comment:
Is this flipped to `true` to let the encoder deal with spaces and other odd formats?

Reply:
Exactly